lib/ansible/modules/network/eos/eos_static_route.py

```diff
@@ -9,7 +9,7 @@
 
 
 ANSIBLE_METADATA = {'metadata_version': '1.1',
-                    'status': ['preview'],
+                    'status': ['deprecated'],
```

Comment:
As part of deprecation process need to rename the module to _eos_static_route.py

Reply:
Done!